src/index.ts

```diff
@@ -40,6 +40,17 @@ export async function main(): Promise<void> {
       ),
     );
 
+    // Assumption: The filepath is of the format `teams/{team-name}` or `teams/{team-name}/migrations` (for db migrations)
+    const team =
+      core.getInput('files').split('/')[1]?.toLowerCase() ?? 'unknown';
+    const isMigration = core.getInput('files').includes('migrations');
+    // Assumption: Environment name is the provided target regexp
+    const env = core.getInput('promotion-target-regexp');
+
+    core.setOutput('team', team);
+    core.setOutput('is-migration', isMigration);
+    core.setOutput('target-env', env);
```

Comment:
It would also be cool if we could get a list of applications that were updated if we move to `update-promoted-values` but that can be a later pr.